app/views/declaration/total_number_of_items.scala.html

```diff
@@ -49,14 +51,25 @@
 
 @hasErrors = @{form.hasErrors || form.hasGlobalErrors}
 
-@input(field: Field, labelKey: String, inputClasses: String) = {
-    @govukInput(Input(
-        id = field.id,
-        name = field.name,
-        value = field.value,
-        label = Label(content = Text(messages(labelKey)), classes = "govuk-label"),
-        classes = s"$inputClasses ${if(field.hasErrors) "govuk-input--error"}"
-    ))
+@input(field: Field, labelKey: String, inputClasses: String, hint: Option[String] = None) = {
+    @if(hint.isDefined) {
+        @govukInput(Input(
+            id = field.id,
+            name = field.name,
+            value = field.value,
+            hint = Some(Hint(content = Text(messages(hint.get)))),
+            label = Label(content = Text(messages(labelKey)), classes = "govuk-label"),
+            classes = s"$inputClasses ${if(field.hasErrors) "govuk-input--error"}"
```

Comment:
@magicaljellybeans it looks like you're setting the error class yourself here but not actually displaying the error message, the Input case class has an errorMessage property that you can use to display an error with the field 

- [template showing how it will be used](https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukInput.scala.html#L50-L64)
- [you also don't need to pass the `govuk-input--error` class in yourself](https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukInput.scala.html#L24)
- [the input case class](https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/input/Input.scala#L35) and the [error message one](https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/errormessage/ErrorMessage.scala#L25-L31)

however if you can, I'd recommend seeing if you can use the `RichInput(Input()).withFormField(play.api.data.Field())` implicit helpers we provide because they help simplify the integrate with play forms

- [the implicit helper showing what it does for you out of the box](https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/scala/uk/gov/hmrc/govukfrontend/views/implicits/RichInputSupport.scala#L37-L42)
- [examples and documentation from our readme for these implicits](https://github.com/hmrc/play-frontend-hmrc#useful-implicits)